sqlitecluster/SQLiteNode.cpp

```diff
@@ -43,7 +43,6 @@
 //                   the leader to confirm that these responses were in fact sent in response to the correct message,
 //                   and not some old out-of-date message from the past.
 // Response:         Sent in STANDUP_RESPONSE, either "approve" or "deny".
-// Reason:           The reason for a failure case (typically, a "deny" Response). Explanatory string.
```

Comment:
why did we strike this field?

Reply:
It's not useful enough to bother maintaining. If you really care about this, the denying node will have logged it anyway, and so if you're looking at logs (which is the only way to see a "reason" field anyway), you can look at the logs of the other node.

Reply:
denying node logs are fine 👍